identity-and-access-management/organizations-and-projects/additional-content/organization-quotas.mdx

```diff
@@ -261,6 +261,40 @@ For example, if you choose to create a container with 512 MB of memory and a max
 The maximum RAM quota is obtained by multiplying the maximum scale factor of your container by the selected RAM quantity.
 For example, if you choose to create a container with 512 MB of memory and a max scale of 20, you will have 10 GB RAM.
 
+
+## Messaging and Queuing
+
+Scaleway Messaging and Queuing is a message broker tool that allows you to transfer messages between different microservices and platforms.
+
+#### Usage limits
+
+The following usage limits are enforced as Organization quotas when using Messaging and Queuing:
+
+**NATS**
+
+| Resource                                       | Criteria   | Limits   | Scope        |
+|------------------------------------------------|------------|----------|--------------|
+| NATS account                                   | Max number | 10       | Organization |
+| NATS message payload                           | Max size   | 512 KB   | Message      |
+| NATS queue/stream storage                      | Max size   | 300 MB** | NATS account |
+| Max number of NATS streams                     | Max number | 10       | NATS account |
+| Max number of NATS consumers                   | Max number | 50       | NATS account |
+
+**This implies stream replication. If you choose to replicate your stream across all nodes (3), a storage limit of 100 MB applies.
```

Comment:
Just to be sure, does `**` render properly ? I saw cases where I had to escape the characters

Reply:
It's a straight copy from https://www.scaleway.com/en/docs/serverless/messaging/reference-content/limitations/ so it should be OK

Reply:
I checked it works